Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update && \
         gnupg \
         help2man \
         libbz2-dev \
-        libcurl4-gnutls-dev \
+        libcurl4-openssl-dev \
```

Comment:
Why is this change needed?

Reply:
For some reason, that change allows @JustinKyleJames to build the image AFAIK. I'm able to build without that change, so it's possible it could be something in his environment. 

@JustinKyleJames Please capture the build output you get into the issue you created. @SwooshyCueb may be able to see the problem.

Reply:
I just added it to that issue.  Also will copy it here:

```
dpkg: dependency problems prevent configuration of irods-runtime:
 irods-runtime depends on libcurl4; however:
  Package libcurl4 is not installed.

dpkg: error processing package irods-runtime (--install):
 dependency problems - leaving unconfigured
Errors were encountered while processing:
 irods-runtime
```